NGitLab/PublicAPI.Unshipped.txt

```diff
@@ -2117,6 +2117,7 @@ NGitLab.Models.Issue.MergeRequestsCount.get -> int
 NGitLab.Models.Issue.MergeRequestsCount.set -> void
 NGitLab.Models.Issue.Milestone -> NGitLab.Models.Milestone
 NGitLab.Models.Issue.MovedToId -> int?
+NGitLab.Models.Issue.References -> NGitLab.Models.References
```

Comment:
@avkranenburg, you're still missing info in this file. Also, this line is not at the correct location; it should be right after the next one, to remain sorted.

If you're using Visual Studio, I would recommend that you
- Undo all changes to the file (i.e. bring it back to the state it's in in the `main` branch)
- Use the IDE's embedded refactoring and select "Fix all occurrences in: **Solution**"

![image](https://github.com/user-attachments/assets/737ac44e-3c30-4e4f-b07f-0fa19865373f)

If you're using VS Code, Rider or something else, I'm not sure how to achieve the same effect.